### PR TITLE
LWS-126: Add search button etc

### PR DESCRIPTION
**Pull request by jesperengstrom**
### Tickets involved
[LWS-126](https://kbse.atlassian.net/browse/LWS-126)

### Summary of changes

* Add a button for the search input
* Use new button classes on existing buttons
* Extend button classes with disabled/active state, negative variant etc.
* Tailwind-ify the CSS a bit
* Add focus outline for the search input

### CODE REVIEW

File: lxl-web/src/lib/components/Search.svelte [hunk 1]
```diff
@@ -34,11 +35,13 @@
...
 
-<form class="w-full" action="find" on:submit={handleSubmit}>
+<form class="relative w-full" action="find" on:submit={handleSubmit}>
 	<!-- svelte-ignore a11y-autofocus -->
 	<input
 		id="main-search"
-		class="h-12 w-full rounded-full border-t-accent-dark/32 text-secondary sm:h-14"
+		class="h-12 w-full rounded-full pr-12 text-secondary shadow-accent-dark/32 focus:shadow-search-focus focus:outline
+			focus:outline-8 focus:outline-accent-dark/8 sm:h-14
+			sm:pr-28"
```
Comment (johanbissemattsson) [inline lxl-web/src/lib/components/Search.svelte:44]: I would prefer the default browser outline styling as the outlines from the Figma sketches are extremely subtle. I feel that we loose more (accessiblity) than we gain from the custom styling.

Current custom outline (which I think is too subtle):
Skärmavbild 2024-05-13 kl  13 41 18

Default browser outline in Firefox:
Skärmavbild 2024-05-13 kl  13 41 01

**Review by johanbissemattsson — COMMENTED** [review]
LGTM (apart from the subtle search input outline)

File: lxl-web/src/lib/components/Search.svelte (hunk 1 above)
Comment (johanbissemattsson) [reply lxl-web/src/lib/components/Search.svelte:44]: I'm also ok with having a shadow outline if has more contrast / is more visible.

Comment (jesperengstrom) [reply lxl-web/src/lib/components/Search.svelte:44]: Skärmavbild 2024-05-13 kl  14 00 36

Revisiting the Figma design now I realize the outline is probably meant as hover state (similarly, the buttons also have hover outline). Maybe a bit unconventional for an input field. I'm ok with skipping it now and going for a general, fully visible/accessible focus ring later.

Comment (jesperengstrom) [reply lxl-web/src/lib/components/Search.svelte]: Turned the contrast up a notch. Visible enough?
Skärmavbild 2024-05-14 kl  10 51 11

Comment (johanbissemattsson) [reply lxl-web/src/lib/components/Search.svelte]: Much better, I'm fine with it :)

The only problem with adding a custom outline in one place is that missing customized outlines on other elements gets more glaring/noticeable. 😀 But that should not hinder us!

**Review by olovy — APPROVED** [review]
LGTM! :ship:

File: lxl-web/src/lib/components/Search.svelte (hunk 1 above)
Comment (jesperengstrom) [reply lxl-web/src/lib/components/Search.svelte]: Yep, but that's the beta feel :)

**Pull request merged**